src/domain/opendatafabric/src/serde/yaml/formats.rs

```diff
@@ -10,6 +10,23 @@
 // TODO: Yep... all this to serde an Option<DateTime> in a slightly different
 // format. See: https://github.com/serde-rs/serde/issues/723
 
+pub mod base64 {
+    use ::base64::Engine;
+    use serde::{Deserialize, Deserializer, Serializer};
+
+    pub fn serialize<S: Serializer>(data: &Vec<u8>, serializer: S) -> Result<S::Ok, S::Error> {
```

Comment:
To encode binary data as base64 in text-based formats like YAML/JSON. Otherwise serde_yaml will output it as an array of numbers